charts/_templates.gotmpl

```diff
@@ -8,6 +8,9 @@ R and Python packages that Data Scientists need to create and share data product
   {{- else if eq .Name "rstudio-workbench" -}}
 Data Scientists use [RStudio Workbench](https://www.rstudio.com/products/workbench/) to analyze data and create data
 products using R and Python.
+  {{- else if eq .Name "posit-chronicle" -}}
+IT Administrators and Business Users use [Posit Chronicle](https://www.rstudio.com/products/chronicle/) to aggregate and monitor
```

Comment:
This url doesn't exist yet and https://www.rstudio.com/products/chronicle will 404, this is a stub for now until we release a suitable doc link

Reply:
I just happened to check this PR out as I saw the message, neat to see!  A nitpick here, but I don't think this will be at "products".  Perhaps have it mimic pro-drivers as the closest example?  https://docs.posit.co/chronicle

Also we should switch these all to posit.co!

Reply:
yeah i was thinking about this, I don't think this really belongs as a product either so maybe I can leave this template alone. https://docs.posit.co/chronicle seems like a good future link too!

> Also we should switch these all to posit.co!
Yeah I definitely agree. I'm hoping at least we can set the right standards with chronicle to help guide the name migration of our other charts - but then chronicle will kindof stick out in the interim.